Extension/YapoSubModule.cs

```diff
@@ -25,25 +27,48 @@ protected override void OnSubModuleLoad()
                 FileDatabase.Initialise(Strings.MODULE_FOLDER_NAME);
                 Settings settings = FileDatabase.Get<Settings>(Settings.InstanceId) ?? new Settings();
                 SettingsDatabase.RegisterSettings(settings);
-                
+
                 UIExtender.Register();
                 new Harmony("YAPO").PatchAll();
-            } catch (Exception exception)
+            }
+            catch (Exception e)
```

Comment:
Why change this to `e`?
`Exception` is clearer. Personally prefer variable names that aren't shortened or abbreviated (except for LINQ predicates)

Reply:
I usually agree with this but in the case of exceptions in a try-catch block I usually shorten them since lines tend to grow long with exception messages and all.

I'll revert it.